packages/flutter_localizations/test/date_picker_test.dart

```diff
@@ -223,6 +223,59 @@ void main() {
 
     await tester.tap(find.text('ANNULER'));
   });
+
+  group('screen configurations', () {
```

Comment:
What these tests have in common seems to be that we're varying locale?

Reply:
I renamed this group to "locale fonts don't overflow layout". Couldn't think of a better description. Let me know if you have one :).